openfisca_core/entities.py

```diff
@@ -9,13 +9,15 @@
 import dpath
 
 from indexed_enums import Enum, EnumArray
-from formulas import ADD, DIVIDE
 from scenarios import iter_over_entity_members
 from simulations import check_type, SituationParsingError
 from holders import Holder, PeriodMismatchError
 from periods import compare_period_size, period as make_period
 from errors import VariableNotFound
 
+ADD = 'add'
```

Comment:
Why not replace ADD for `'add'` in the code ? (same for DIVIDE)

Reply:
`ADD` is a constant imported in legislation variables files, and used for calls like `person('salary', year, options = [ADD])`. It is not just used for this file.

Reply:
Anyway, in general, constants are preferred to re-defining the same string _n_ times. It is more robust to typos, more readable, and a little more efficient performance-wise.